### DISCUSSION TITLE
Error setting value '../build/app/outputs/bundle/release/app.aab' for option 'aab',     You passed invalid parameters to 'upload_to_play_store'.

### DISCUSSION THREAD

**Discussion by gikwegbu** [fastlane/fastlane#21095] [Q&A] [answered]
Having issues with fastlane upload_to_app_store. 
Found out the error was from the path to the .aab file

**Comment by slavdok** [comment, marked as answer]
Are you sure the file exists?
Try running command `file ../build/app/outputs/bundle/release/app.aab` from the same place you invoke fastlane

**Comment by gikwegbu** [comment]
Hi @slavdok , thanks for reaching out.
I checked the file location, and it was my fault. 

For future reference,
developer should run
`$ flutter build appbundle`
copy the path printed on the console to be sure

This question has been answered. Thanks
